### PR TITLE
Add internal repository configuration

### PR DESCRIPTION
**Pull request by jrodbx**
Makes deploying builds to internal repositories a bit easier

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -122,6 +122,18 @@ subprojects {
...
+        /**
+         * Are you a Square employee?  If so, then set the following properties in
+         * ~/.gradle/gradle.properties to enable pushing to our internal repository
+         *
+         * artifactoryUsername=YOUR_LDAP
+         * artifactoryPassword=YOUR_API_KEY
+         */
+        maven {
+          name = "artifactory"
+          url = "https://maven.global.square/artifactory/releases"
```
Comment (JakeWharton) [inline build.gradle:134]: We're not supposed to leak internal URLs. Can you use an environment variable? For Treehouse we use `-PRELEASE_REPOSITORY_URL="the url"` on the CLI to release internally.

Comment (jrodbx) [reply build.gradle]: ok, addressed by generalizing to any internal repository.  PTAL!

File: gradle.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 GROUP=app.cash.paparazzi
-VERSION_NAME=0.9.0-SNAPSHOT
+VERSION_NAME=0.9.0-alpha09
```
Comment (JakeWharton) [inline gradle.properties:2]: Revert?

**Review by JakeWharton — APPROVED** [review]

File: gradle.properties (hunk 2 above)
Comment (jrodbx) [reply gradle.properties:2]: oops lol

**Pull request merged**